apps/manager-client/src/styles.css

```
@@ -0,0 +1,347 @@
+  border-radius: 12px;
+  background: #1d2e45;
+  color: #e8efff;
+}
+
+.page-wrap {
+  width: min(1080px, calc(100% - 2rem));
+  margin-inline: auto;
+}
+
+.display-title {
+  font-family: 'Fraunces', Georgia, serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)